src/webapp/components/SendSmsDialog.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+import useAxios from "axios-hooks";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { CircularProgress } from "@material-ui/core";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import TextareaAutosize from "@material-ui/core/TextareaAutosize";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Button from "@material-ui/core/Button";
+import { useTranslation } from "react-i18next";
+import TextField from "@material-ui/core/TextField";
+import sharedStylesFn from "../style/sharedStyles";
+
+const useStyles = makeStyles(theme => ({
+  ...sharedStylesFn(theme),
+  messageTextarea: {
+    width: "100%"
+  }
+}));
+
+const SendSmsDialog = ({ message, isOpen, onClose }) => {
+  const { t: str } = useTranslation();
+  const [smsMessage, setSmsMessage] = useState(message);
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [invalidPhoneNumber, setInvalidPhoneNumber] = useState(false);
+
+  const classes = useStyles();
+
+  const [{ data, loading, error }, submit] = useAxios(
+    {
+      url: `/api/send-sms`,
+      method: "post"
+    },
+    { manual: true } // Don't send on render
+  );
+
+  useEffect(() => {
+    if (data) {
+      onClose();
+    }
+  }, [data, onClose]);
+
+  const validatePhoneNumber = number => {
+    const phoneNumberRegEx = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/;
+    if (number.match(phoneNumberRegEx)) {
+      setInvalidPhoneNumber(false);
+      return true;
+    }
+    setInvalidPhoneNumber(true);
+    return false;
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    if (validatePhoneNumber(phoneNumber)) {
+      submit({
+        data: {
+          smsMessage,
+          phoneNumber
+        }
+      });
+    }
+  };
+
+  return (
+    <Dialog
+      fullWidth
+      open={isOpen}
+      onClose={onClose}
+      aria-labelledby="sms-dialog"
+    >
+      <DialogTitle id="sms-dialog-title">Send SMS</DialogTitle>
+      <DialogContent>
+        <DialogContentText>
+          {str("webapp:sendSms.messageInfo")}
+          <TextareaAutosize
```

Comment:
So no default currently? It would be great to reach out to wg_education and wg_intake with a document of resources or something similar to send

Reply:
Cool - I posted in wg_intake to see if they have thoughts on if the email template message could be updated or altered for the SMS. All other feedback addressed, thanks!

Reply:
i think intake now has a default on this - let me know how i can best get this copy to you!

Reply:
ooo awesome! Slack or jorganisak@gmail.com works